docs/gallery/External_Resources.py

```diff
@@ -37,24 +37,30 @@
 # as a string, but if you want to reference a key then you reference
 # the key object itself.
 
-data = Data(name="species", data='homo sapien')
-er.add_ref(container=data, field='Data', key='human', resource_name='NCBI_Taxonomy',
+data = Data(name="species", data=['Homo sapien', 'Mus Musculus'])
```

Comment:
Minor but important to get right:
```suggestion
data = Data(name="species", data=['Homo sapiens', 'Mus musculus'])
```

Reply:
here and throughout the rest